### DISCUSSION TITLE
Game that spans multiple screens.

### DISCUSSION THREAD

**Discussion by SynergyJFernald** [stride3d/stride#2179] [Q&A]
Hello, 

I'm working on a project that can span up to 4 monitors. The only ways I've seen to do this in stride is similar to a split screen setup.  Where in Unity I could Set Camera 1 to Display 1, Camera 2 to Display 2, Camera 3 to Display 3, and camera 4 to Display 4.  Is there a better way to accomplish this than split screen?

Thanks for any assistance.

**Comment by tebjan** [comment]
Have a look at: https://doc.stride3d.net/4.2/en/manual/graphics/graphics-compositor/render-textures.html
